### DISCUSSION TITLE
how chage color

### DISCUSSION THREAD

**Discussion by andriux26** [jekhokie/raspberry-noaa-v2#433] [General]
need change color sheme for meteor foto

**Comment by andriux26** [comment]
![Be_pavadinimo](https://user-images.githubusercontent.com/5492430/130422997-9c0a7cca-ae59-4870-bc17-eab64316c422.jpg)

**Reply by ve3elb** [reply]
In scripts/image processors 

Meteor false color script last line change the RGB Colors. 

 -r 65 -g 65 -b 64 -d

I will double check but I think I changed it to:
 -r 66 -g 65 -b 64 -d

**Reply by andriux26** [reply]
thank you

**Reply by andriux26** [reply]
ok at now

**Comment by andriux26** [comment]
2 to 1

**Reply by dom-robinson** [reply]
I can recall solving this before but not how! @colinluthier do you remember?) 

Do you know which demodulator you are using? 
In the settings you will see either ```meteor_receiver: 'gnuradio'``` or ```meteor_receiver: 'rtl_fm'```

Try flipping them - that may sort it out...

**Reply by colinluthier** [reply]
With RTL-FM I get the first one in the morning and the second in the evening. It has to do with what sensors you pick to decode.

**Reply by andriux26** [reply]
How find to look demoduliator
